table/cuckoo/cuckoo_table_reader.cc

```diff
@@ -186,7 +186,10 @@ Status CuckooTableReader::Get(const ReadOptions& /*readOptions*/,
             return s;
           }
           bool dont_care __attribute__((__unused__));
-          get_context->SaveValue(found_ikey, value, &dont_care);
+          get_context->SaveValue(found_ikey, value, &dont_care, &s);
```

Comment:
Should we check the return value from `SaveValue()` ?

Reply:
I haven't thought about it. Do you have an explanation of what can go wrong without it and/or proposal of what to do with it?

Reply:
I think we should check the return value as is done in the vicinity of this code.